packages/components/src/modal/index.tsx

```diff
@@ -283,7 +289,17 @@ function UnforwardedModal(
 								) }
 							</div>
 						) }
-						<div ref={ childrenContainerRef }>{ children }</div>
+						<div ref={ childrenContainerRef }>
```

Comment:
Trying to combine the focus ref with the `childrenContainerRef` results in errors due to the `useEffects` above. Any ways to avoid the extra div appreciated.

Reply:
`useMergeRefs` can help.
```jsx
						<div
							ref={
								useMergeRefs( [
									childrenContainerRef,
									focusOnMount === 'firstContentElement'
										? focusOnMountRef
										: null
							] }
						>
							{ children }
						</div>
```

Reply:
Ah nice. Apply at the point of use. That's great. Thank you 🙇

Reply:
I was also going to suggest using `useMergeRefs`, thank you @stokesman !